packages/api/src/models/managementPlanGroupModel.js

```diff
@@ -0,0 +1,65 @@
+/*
+ *  Copyright (C) 2023 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+import { Model } from 'objection';
+import baseModel from './baseModel.js';
+import ManagementPlan from './managementPlanModel.js';
+
+class ManagementPlanGroup extends baseModel {
+  static get tableName() {
+    return 'management_plan_group';
+  }
+
+  static get idColumn() {
+    return 'management_plan_group_id';
+  }
+
+  async $beforeInsert(context) {
+    await super.$beforeInsert(context);
+  }
+
+  async $beforeUpdate(opt, context) {
+    await super.$beforeUpdate(opt, context);
+  }
+
+  static get jsonSchema() {
+    return {
+      type: 'object',
+      required: ['repetition_count', 'repetition_config'],
+      properties: {
+        management_plan_group_id: { type: 'string' },
+        repetition_count: { type: 'integer' },
+        repetition_config: { type: 'string' },
```

Comment:
Since we specified `jsonb` for that column in the migration should it be `repetition_config: { type: 'object' }`?

Reply:
I was thinking on a `JSON.stringify()` version of the config. But looking at other instances of `jsonb` you're right.